### PR TITLE
logic to display/choose pii for property types

### PR DESCRIPTION
**Pull request by katherinebernstein**
add PII as option when creating property type, show PII in property type list

### CODE REVIEW

File: src/views/Main/Schemas/Components/NewEdmObjectInput.js [hunk 1]
```diff
@@ -222,13 +224,38 @@ export class NewEdmObjectInput extends React.Component {
...
+            onChange={this.handlePiiChange}
+            options={options}
+            className={styles.piiSelect} />
+        <div className={styles.spacerSmall} />
+      </div>
+    );
+  }
+
+  handlePiiChange = (e) => {
+    const newValue = (e && e !== undefined) ? e.value : false;
```
Comment (UnsungHero97) [inline src/views/Main/Schemas/Components/NewEdmObjectInput.js]: what are the possible values for `pii`? is it just a boolean?

Comment (UnsungHero97) [inline src/views/Main/Schemas/Components/NewEdmObjectInput.js]: `(e && e !== undefined)` seems redundant. what are you trying to check for specifically?

Comment (katherinebernstein) [inline src/views/Main/Schemas/Components/NewEdmObjectInput.js]: oops yeah it is.

Comment (katherinebernstein) [inline src/views/Main/Schemas/Components/NewEdmObjectInput.js]: yeah

**Pull request merged**